common/djangoapps/static_replace/management/commands/clear_collectstatic_cache.py

```diff
@@ -1,14 +1,21 @@
-###
-### Script for importing courseware from XML format
-###
+"""
+Script for importing courseware from XML format
+"""
 
-from django.core.management.base import NoArgsCommand
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
 from django.core.cache import caches
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
+    """
+    Implementation of the management command
+    """
+
     help = 'Import the specified data directory into the default ModuleStore'
 
-    def handle_noargs(self, **options):
+    def handle(self, *args, **_):
```

Comment:
Is the trailing underscore a typo?

Reply:
It's not. There are kwargs passed in from the management command infrastructure that we don't care about, but still need to accept. The underscore convention allowed this without having to explicitly silence pylint, but I was on the fence about it and could go either way. (cc @doctoryes)